source/patient/valueset-pronouns.xml

```diff
@@ -22,26 +27,26 @@
   <compose>
     <include>
       <system value="http://hl7.org/fhir/pronouns"/>
-	  <concept>
-		<code value="he-him"/>
-		<display value="He, Him, His, Himself"/>
-	  </concept>
-	  <concept>
-		<code value="she-her"/>
-		<display value="She, Her, Hers, Herself"/>
-	  </concept>
-	  <concept>
-		<code value="they-them"/>
-		<display value="They, Them, Their, Theirs, Themself"/>
-	  </concept>
-	  <concept>
-		<code value="other"/>
-		<display value="Other"/>
-	  </concept>
-	  <concept>
-		<code value="unknown"/>
-		<display value="Unknown"/>
-	  </concept>
+      <concept>
```

Comment:
@JohnMoehrke are these added here intentionally? they don't seem to be in the scope of this ticket.

Reply:
Some files get updated by the build process. When git notices these have changed, you are expected to include them. This is just part of the process of maintaining the fhir core.

Reply:
yes, sometimes one finds that they change for reasons outside your scope. This is likely due to the previous person not submitting them properly... we all must work as a team.